packages/fxa-auth-server/lib/senders/email.js

```diff
@@ -658,6 +658,7 @@ module.exports = function (log, config, bounces) {
         flowId,
         flowBeginTime,
         deviceId,
+        accountVerified,
```

Comment:
When this function is called, do we know that `accountVerified` is added to `message`?